### ISSUE TITLE
Some functions of the dropdown menu don't work

### ISSUE THREAD

**Issue by sibelius1269** [orgzly-revived/orgzly-android-revived#59]
Hey. Excellent job. I'm really glad to know that you guys have taken on the effort to continue with the updates for this great app. Thanks a lot and keep it up!

The icons in the drop-down menu for queries don't work, neither the one for direct note deletion nor the one for changing its location in a notebook. Can you fix it in the next stable version? Thanks a lot.

**Comment by alensiljak** [comment]
Do you mind providing a screenshot? Which menus do you mean?

**Comment by sibelius1269** [comment]
Of course.

**Comment by sibelius1269** [comment]
![Screenshot_2023-11-13-21-58-57-85_ed2cde8a5c8dee394aae50c5114b3a51](https://github.com/orgzly-revived/orgzly-android-revived/assets/39271278/9cbf33cb-d51a-46be-89b9-f8c862be94f9)
It's possible that the image I attached in my email response may not have been viewed, so I'm attaching it here.

Out of the three accesses shown in screenshot, only the 'go to note' option is working.

**Comment by sibelius1269** [comment]
Yeah, for me, it only happens on the search screen too. And yeah, I'm using version 1.8.14 for sure. My smartphone is a Realme 9 Pro + with Android 13
